packages/contextual-help/stories/contextual-help.stories.ts

```diff
@@ -0,0 +1,171 @@
+import { spreadProps } from '../../../test/lit-helpers.js';
+import type { Placement } from '@spectrum-web-components/overlay/src/overlay-types.js';
+
+interface StoryArgs {
+    label: string;
+    variant: 'info' | 'help';
+    headlint: string;
+    placement: Placement;
+    [key: string]: unknown;
+}
+
+export default {
+    title: 'Contextual Help',
+    component: 'sp-contextual-help',
+    args: {
+        label: '',
+        variant: 'info',
+        headline: undefined,
+        placement: undefined,
+    },
+    argTypes: {
+        variant: {
+            name: 'variant',
+            type: { name: 'string', required: false },
+            description: '',
+            table: {
+                defaultValue: { summary: 'info' },
+            },
+            options: ['info', 'help'],
+            control: {
+                type: 'radio',
+            },
+        },
+        label: {
+            name: 'label',
+            type: { name: 'string', required: false },
+            table: {
+                type: { summary: 'label' },
+                defaultValue: { summary: `'Help' | 'Info'` },
+            },
+            control: 'text',
+        },
+        headline: {
+            name: 'headline',
+            type: { name: 'string', required: false },
+            table: {
+                type: { summary: 'headline' },
+                defaultValue: { summary: '' },
+            },
+            control: 'text',
+        },
+        placement: {
+            name: 'placement',
+            type: { name: 'string', required: false },
+            description:
+                'The placement of the popover content in relation to the button',
+            table: {
+                type: { summary: 'string' },
+                defaultValue: { summary: 'bottom-start' },
+            },
+            control: {
+                type: 'select',
+                labels: {
+                    top: 'top',
+                    'top-start': 'top-start',
+                    'top-end': 'top-end',
+                    right: 'right',
+                    'right-start': 'right-start',
+                    'right-end': 'right-end',
+                    bottom: 'bottom',
+                    'bottom-start': 'bottom-start',
+                    'bottom-end': 'bottom-end',
+                    left: 'left',
+                    'left-start': 'left-start',
+                    'left-end': 'left-end',
+                },
+            },
+            options: [
+                'top',
+                'top-start',
+                'top-end',
+                'right',
+                'right-start',
+                'right-end',
+                'bottom',
+                'bottom-start',
+                'bottom-end',
+                'left',
+                'left-start',
+                'left-end',
+            ],
+        },
+    },
+};
+
+export const Default = (args: StoryArgs): TemplateResult => {
+    return html`
```

Comment:
We need to update our generated files, but can you move this to match https://github.com/adobe/spectrum-web-components/blob/main/packages/accordion/stories/template.ts

Then as much sharing of `Template` as you can between the stories would be great.